docs/source/API/core/macros-special/host_device_macros.rst

```diff
@@ -0,0 +1,141 @@
+
+``Function markup macros``
```

Comment:
Where does the "markup" term come from?
Should we be saying "annotation" instead?